demos/bubble_shear.py

```diff
@@ -0,0 +1,237 @@
+# Advection of a tracer bubble in a non-uniform shear flow
+# ===================================================
+
+# In this demo, we consider the advection of a scalar concentration field, similarly to
+# our previous solid body rotation demo. However, we now consider a background velocity
```

Comment:
This demo will follow on from the Gray-Scott demo so please provide references between this demo and that one so that the demos can be read through in order.

Reply:
Fixed in 262c78a51a99714a4794b55259edf6fc5aea3567.